lib/configure.js

```diff
@@ -254,18 +254,21 @@ function configure (gyp, argv, callback) {
     //  - the out/Debug directory
     //  - the root directory
     var node_exp_file = undefined
-    if (process.platform === 'aix') {
+    if (process.platform === 'aix' || process.platform === 'os390') {
+      var ext = process.platform === 'aix' ? 'exp' : 'x'
       var node_root_dir = findNodeDirectory()
-      var candidates = ['include/node/node.exp',
-                        'out/Release/node.exp',
-                        'out/Debug/node.exp',
-                        'node.exp']
+      var candidates = ['include/node/node',
+                        'out/Release/node',
+                        'out/Debug/node',
+                        'node'].map(function(file) {
+                          return `${file}.${ext}`
```

Comment:
No backticks.

Reply:
Sorry, i missed that one!

Reply:
Why no backticks? `node-gyp@4` is node>=4.
https://github.com/nodejs/node-gyp/blob/7245415296c5f0a97e10dba6dafbaf2332a67578/package.json#L38-L40

Reply:
Well, that was a surreptitious change.  I wouldn't have hidden that in an otherwise unrelated commit.  (I'm talking about 5f924ce6.)

At any rate, this way we can do another 3.x release and include this change.

Reply:
Sure, no harm in keeping compatibility.